api/src/zipcode/scorecard/get.handler.ts

```diff
@@ -19,10 +19,10 @@ async function getZipCodeData(
     resourceArn: process.env.RESOURCE_ARN ?? '',
     schema: SCHEMA,
     secretArn: process.env.CREDENTIALS_SECRET ?? '',
-    sql: `SELECT census_geo_id as zip_code,
-                 average_home_age,
-                 average_income_level,
-                 average_social_vulnerability
+    sql: `SELECT census_geo_id,
+                 home_age_index,
```

Comment:
do we want to show indices or just median home age / income?

Reply:
Added both indices are actual values. Let me know if you want me to add anything else!